components/sup/src/main.rs

```diff
@@ -497,7 +498,7 @@ fn ui() -> UI {
 /// populates all *shared* options between `run` and `load`.
 fn update_svc_load_from_input(m: &ArgMatches, msg: &mut protocol::ctl::SvcLoad) -> Result<()> {
     msg.bldr_url = Some(bldr_url(m));
-    msg.bldr_channel = Some(channel(m));
+    msg.bldr_channel = Some(channel(m).to_string());
```

Comment:
Can `SvcLoad` handle converting a `ChannelIdent` into a string when it gets converted into a Protobuf? That'd help to keep the "string"ness of channels at the boundaries of the system, while keeping only `ChannelIdent` in the interior.

Reply:
Good idea, added in https://github.com/habitat-sh/habitat/pull/6074/commits/cbebb204f2ee3b0b2cafe139cbf99ed1baf73d94